### ISSUE TITLE
Should quit a job that makes(forces) me to work on ember 2.2 in 2018?

### ISSUE THREAD

**Issue by rajatvijay** [kentcdodds/ama#434]
The main issues I see here are: 

1. Most of the times I have to solve the problems already solved by the react community (though for react).
2. I have a very strict timeline for delivery which stops me from understanding the framework and enjoys its benefits(if there are any as compared to other frameworks in 2018).
3. I might not be learning anything new.
4. It might not be helping me in advancing my career since I am working on a tech that is already dead.

**Comment by hedgerh** [comment]
If you aren't happy and feel your job isn't serving your long term career goals, maybe it is time to look for new opportunities.

**Comment by hedgerh** [comment]
Kent likely won't get around to answering this.  I would recommend checking out the Discord community Reactiflux.  They have a #jobs-advice channel that could be helpful to you.

Okay if I go ahead and close this?

**Comment by kentcdodds** [comment]
Actually, this is the kind of question I'm happy to answer. Please leave this one open.

**Comment by kentcdodds** [comment]
Hi! I'm afraid I don't have time to give a super solid answer to all my open questions right now, so I just went through and answered all my AMA questions in the form of a video: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u

A few things that folks can do to help make things better for others reading the questions and answers:

1. Link to when I start answering the question. Add `&t=xxmxxs` to the end of the URL like this: https://www.youtube.com/watch?v=y23g8L8UtoQ&list=PLV5CVI1eNcJgCrPH_e6d57KRUTiDZgs0u&t=2m5s
2. Comment with links I referenced
3. Summarize what I talk about

Thanks!
